src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/BuildStatusNotifier.java

```diff
@@ -210,6 +211,21 @@ private Map createReportMap(Result result) {
         return externalReport;
     }
 
+    private Map createReportData(String fieldName, String type, String value){
+        Map map = new HashMap();
+        map.put("fieldName", fieldName);
+        map.put("type", type);
+        map.put("value", value);
+        return map;
+    }
+
+    private String toStringDuration(long duration){
+        long hours = TimeUnit.MILLISECONDS.toHours(duration) % 24;
+        long minutes = TimeUnit.MILLISECONDS.toMinutes(duration) % 60;
```

Comment:
A. Seems a bit risky. What happens if thats 36 hours? How will you represent it?
B. Please move it to another class which will handle this formatting as it is not part of the BuildStatusNotifier.java.
C. Add tests.